tests/testthat/test-MltplxObject.R

```diff
@@ -1,6 +1,9 @@
 exp <- build_mltplx_exp(200,seed=2025)
 exp <- update_intensity(exp,ps=2,bw=3)
-exp <- update_dist(exp,cor)
+# setting symmetric to FALSE produces the type of distance matrix that
+# this code uses to check for duplicates, i.e. a redundant/upper-lower
+# distance matrix
+exp <- update_dist(exp,cor, symmetric = FALSE)
```

Comment:
this doesn't seem like expected behavior to me - why isn't symmetric set to TRUE in this case?

Reply:
Sorry, meant to respond to both of these yesterday. 

We can probably remove the line, just want to check with Maria first.

The reason symmetric is set to FALSE there is because the code that checks for successful removal of duplicates expects a symmetric matrix. That code might be redundant at this point, but for now, I just said the distance metric was asymmetric so the test would receive the input it was expecting.